Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -3472,7 +3473,11 @@ else if (thisReq_ instanceof ServerRequestLogout) {
                 } catch (JSONException ex) {
                     ex.printStackTrace();
                 }
+                if (thisReq_ instanceof ServerRequestInitSession) {
+                    branchUniversalReferralInitWrapper_ = null;
```

Comment:
If we nullify the wrapper on response, then, when the app is opened again and the Application detects a new Activity added to the stack, there's a risk that the callback won't be referenced. Isn't that right?